### PR TITLE
misc: fix typo in performance-budget smoke test

### PR DESCRIPTION
**Pull request by connorjclark**
this smoke test wasn't reading the correct parameter off ArrayBuffer. nor was the class correct for the stylesheet to apply, so now that it does I updated the font expectation to comply

trivial and silly, i know

### CODE REVIEW

File: lighthouse-cli/test/smokehouse/perf/expectations.js [hunk 1]
```diff
@@ -72,11 +72,11 @@ module.exports = [
       audits: {
         'resource-summary': {
           score: null,
-          displayValue: '11 requests • 164 KB',
```
Comment (patrickhulce) [inline lighthouse-cli/test/smokehouse/perf/expectations.js]: for anyone else scratching their head how in the world this was affected by the fixture change when `.corsfont` doesn't appear anywhere else in `load-things.html`, the CSS rule is defined over in `cors-fonts.css` :)

File: lighthouse-cli/test/smokehouse/perf/expectations.js [hunk 2]
```diff
@@ -94,19 +94,19 @@ module.exports = [
...
+                sizeOverBudget: '52000±1000',
               },
               {
                 resourceType: 'script',
                 countOverBudget: '2 requests',
                 sizeOverBudget: '25000±1000',
               },
-              {
-                resourceType: 'font',
-                countOverBudget: undefined,
```
Comment (brendankenny) [inline lighthouse-cli/test/smokehouse/perf/expectations.js]: the intention was to make sure we got a good mixture of the combinations of defined and undefined for these entries, so ideally this wouldn't change from `undefined`.

Maybe the "Undefined items" comment above could be changed to be a little more strongly worded to warn about that and we could change `corsfont` to something completely different so it doesn't look like a bug that it's not getting the font?

File: lighthouse-cli/test/fixtures/perf/perf-budgets/load-things.html [hunk 3]
```diff
@@ -30,7 +30,7 @@
         // Load file as 'other' resource.
         const response = await fetch('http://localhost:10503/preload.html');
         const ab = await response.arrayBuffer();
-        document.querySelector('#corsfont').textContent = ab.length;
+        document.querySelector('.corsfont').textContent = ab.byteLength;
```
Comment (brendankenny) [inline lighthouse-cli/test/fixtures/perf/perf-budgets/load-things.html]: ha, good catch. Clearly QA was lying down on the job for this page :)

File: lighthouse-cli/test/fixtures/perf/perf-budgets/load-things.html [hunk 4]
```diff
@@ -30,7 +30,7 @@
         // Load file as 'other' resource.
         const response = await fetch('http://localhost:10503/preload.html');
         const ab = await response.arrayBuffer();
-        document.querySelector('#corsfont').textContent = ab.length;
```
Comment (patrickhulce) [inline lighthouse-cli/test/fixtures/perf/perf-budgets/load-things.html]: this change is just about text and doesn't matter at all right?

File: lighthouse-cli/test/smokehouse/perf/expectations.js (hunk 1 above)
Comment (brendankenny) [reply lighthouse-cli/test/smokehouse/perf/expectations.js]: my fault, it was sloppy to change it only a little and then draw a line in the sand for the budgets at that point :)

No memory of why I didn't change the classname itself from what's in `fonts.html`, just operating as a human fuzzer at the time, I think :)

Comment (patrickhulce) [reply lighthouse-cli/test/smokehouse/perf/expectations.js]: oh I'm still not clear why we need to use it at all no harm no foul I was just baffled how his text change could result in fonts getting requested 😆 

I think I was just slow on the uptake that this was modeled after another file

File: lighthouse-cli/test/fixtures/perf/perf-budgets/load-things.html (hunk 4 above)
Comment (connorjclark) [reply lighthouse-cli/test/fixtures/perf/perf-budgets/load-things.html]: idk, would the font be loaded if there were no text?

**Comment by connorjclark** [comment]
> I'm not sure why we need to do any of this, but seems harmless :)

I was testing something entirely unrelated, and chose this fixture as my playground. That's how it jumped out at me :)

File: lighthouse-cli/test/smokehouse/perf/expectations.js (hunk 2 above)
Comment (connorjclark) [reply lighthouse-cli/test/smokehouse/perf/expectations.js]: > change corsfont to something completely different so it doesn't look like a bug that it's not getting the font?

yea, sounds good i'll do that and revert all these expectation changes

File: lighthouse-cli/test/smokehouse/perf/expectations.js [hunk 5]
```diff
@@ -110,7 +110,7 @@ module.exports = [
               {
                 resourceType: 'document',
                 countOverBudget: '1 request',
-                sizeOverBudget: '1100±50',
+                sizeOverBudget: '1150±50',
```
Comment (connorjclark) [inline lighthouse-cli/test/smokehouse/perf/expectations.js]: added just enough bytes to the main doc to require this change

File: lighthouse-cli/test/fixtures/perf/perf-budgets/load-things.html (hunk 4 above)
Comment (patrickhulce) [reply lighthouse-cli/test/fixtures/perf/perf-budgets/load-things.html]: oh, oh I see what I'm missing now sorry :) 

1. `new ArrayBuffer().length === undefined`, not just a semantic difference like in node
2. `el.textContent = undefined` clears all textContent, and does not do some sort of string coercion to `"undefined"`

Comment (brendankenny) [reply lighthouse-cli/test/fixtures/perf/perf-budgets/load-things.html]: > el.textContent = undefined clears all textContent, and does not do some sort of string coercion to "undefined"

just to be clear, while the actual behavior is unintended and pointless, the intended behavior was also pointless. There's no reason to add this to the page at all, just making sure (poorly, it turns out) the thing is actually loaded and used :)

**Pull request merged**